docs/content/Metrics.md

```diff
@@ -3,11 +3,182 @@ layout: doc_page
 ---
 # Druid Metrics
 
-Druid emits a variety of query, ingestion, and coordination metrics.
+Druid generates metrics related to queries, ingestion, and coordination.
 
-Metrics can be displayed in the runtime log file or over http (to a service such as Apache Kafka). Metric emission is disabled by default.
+Metrics are emitted as JSON objects to a runtime log file or over HTTP (to a service such as Apache Kafka). Metric emission is disabled by default.
+
+All Druid metrics share a common set of fields:
+
+* `timestamp` - the time the metric was created
+* `metric` - the name of the metric
+* `service` - the service name that emitted the metric
+* `host` - the host name that emitted the metric
+* `value` - some numeric value associated with the metric
+
+Each metric also has their own set of dimensions that vary from metric to metric.
```

Comment:
i was hoping to do this in later PRs and make progress here first. The list of dimensions can wildly vary from metric to metric.